src/components/pages/Memos/Memo.less

```diff
@@ -0,0 +1,44 @@
+.memoModal {
+  .ant-radio-inner::after {
```

Comment:
I like we chose to go with less. I'm a big fan. However, I'm curious about the choice given that our global stylesheet in `/styles/styles.css` is a `.css` file? Doesn't this go against our current pattern?

Reply:
I think converting our css files to less files will help any further conflicts like this from happening. I'd love to hear your thoughts.

Reply:
I went and resolved the merged conflicts inline 💯